packages/Popover/src/positionUtils.spec.js

```diff
@@ -0,0 +1,1050 @@
+          expect(pos.transform).toBe('translate3d(-5px, 0, 0) scale(0.8)');
+        });
+      });
+      describe('Align.Bottom', () => {
+        test('Justify.Start works', () => {
+          const pos = calculatePosition({
+            spacing: SPACING,
+            windowHeight: WINDOWHEIGHT,
+            windowWidth: WINDOWWIDTH,
+            useRelativePositioning: false,
+            align: Align.Bottom,
+            justify: Justify.Start,
+            referenceElPos: refElPos.left,
+            contentElPos: contentElPos,
+          });
+
+          expect(pos.top).toBe(60);
+          expect(pos.left).toBe(0);
+          expect(pos.transformOrigin).toBe('left top');
+          expect(pos.transform).toBe('translate3d(0, -5px, 0) scale(0.8)');
+        });
+
+        test('Justify.Middle repositions to Justify.Start based on available space', () => {
+          const pos = calculatePosition({
+            spacing: SPACING,
+            windowHeight: WINDOWHEIGHT,
+            windowWidth: WINDOWWIDTH,
+            useRelativePositioning: false,
+            align: Align.Bottom,
+            justify: Justify.Middle,
+            referenceElPos: refElPos.left,
+            contentElPos: contentElPos,
+          });
+
+          expect(pos.top).toBe(60);
+          expect(pos.left).toBe(0);
+          expect(pos.transformOrigin).toBe('left top');
+          expect(pos.transform).toBe('translate3d(0, -5px, 0) scale(0.8)');
+        });
+
+        test('Justify.End repositions to Justify.Start based on available space', () => {
+          const pos = calculatePosition({
+            spacing: SPACING,
+            windowHeight: WINDOWHEIGHT,
+            windowWidth: WINDOWWIDTH,
+            useRelativePositioning: false,
+            align: Align.Bottom,
+            justify: Justify.End,
+            referenceElPos: refElPos.left,
+            contentElPos: contentElPos,
+          });
+
+          expect(pos.top).toBe(60);
+          expect(pos.left).toBe(0);
+          expect(pos.transformOrigin).toBe('left top');
+          expect(pos.transform).toBe('translate3d(0, -5px, 0) scale(0.8)');
+        });
+      });
+
+      describe('Align.Left', () => {
+        test('Align.Left repositions to Align.Right based on available space', () => {
+          const pos = calculatePosition({
+            spacing: SPACING,
+            windowHeight: WINDOWHEIGHT,
+            windowWidth: WINDOWWIDTH,
+            useRelativePositioning: false,
+            align: Align.Left,
+            justify: Justify.Start,
+            referenceElPos: refElPos.left,
+            contentElPos: contentElPos,
+          });
+
+          expect(pos.top).toBe(45);
+          expect(pos.left).toBe(15);
+          expect(pos.transformOrigin).toBe('left top');
+          expect(pos.transform).toBe('translate3d(-5px, 0, 0) scale(0.8)');
+        });
+      });
+    });
+
+    describe('when reference element is in the ceneter', () => {
```

Comment:
Let's also do some 'only works' tests when useRelativePositioning is true. Can just do it for centered, don't need to replicate everywhere.